software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -76,6 +76,10 @@
     @CatalogConfig(label = "Stop Command", priority=1)
     ConfigKey<String> STOP_COMMAND = VanillaSoftwareProcess.STOP_COMMAND;
 
+    @CatalogConfig(label = "Suppress failures occurred during installation. When using this parameter, Apache Brooklyn relies that your blueprint will report failures on the check running phase.")
+    ConfigKey<Boolean> SUPPRESS_INSTALL_FAILURE = ConfigKeys.newBooleanConfigKey("install.windows.suppressFailure", "Suppress any installation failure. It is a common case to restart windows together with the installation process." +
+            " In such case, it's hard to tell when the install command finished and entered into restart and you may want to support connectivity or other failures.", false);
```

Comment:
Put the `, false` on a new line. Otherwise someone reading through the code will miss it (unless they scroll).